common-model/src/main/java/com/sequenceiq/common/api/telemetry/doc/TelemetryModelDescription.java

```diff
@@ -14,6 +15,11 @@
     public static final String TELEMETRY_LOGGING_GCS_ATTRIBUTES = "telemetry - logging gcs attributes";
     public static final String TELEMETRY_LOGGING_CLOUDWATCH_ATTRIBUTES = "telemetry - logging cloudwatch attributes";
     public static final String TELEMETRY_LOGGING_STORAGE_LOCATION = "telemetry - logging storage location / container";
+    public static final String TELEMETRY_BACKUP_S3_ATTRIBUTES = "telemetry - backup s3 attributes";
```

Comment:
is it telemetry or just backup? (question stands for all below)

Reply:
Thanks! removed it as it is a typo.